src/main/java/rpgcharacters/userflow/CharacterMenu.java

```diff
@@ -53,88 +70,114 @@ private String printChars () {
+                        results.getInt("race.power_mod") +
+                        results.getInt("archetype.power_mod");
+            int proficiency = results.getInt("proficiency") +
+                              results.getInt("race.proficiency_mod") +
+                              results.getInt("archetype.proficiency_mod");
+            int personality = results.getInt("personality") +
+                              results.getInt("race.personality_mod") +
+                              results.getInt("archetype.personality_mod");
+            int perception = results.getInt("perception") +
+                             results.getInt("race.perception_mod") +
+                             results.getInt("archetype.perception_mod");
+            int experience = results.getInt("experience");
+
+            // format story
+            if (story.length() > 35) {
+                String[] tokens = story.split(" ");
+                story = "";
+                int curLineLen = 0;
+                for (String tok : tokens) {
+                    if (curLineLen == 0) {
+                        story += "\n\t";
+                    }
+                    story += tok;
+                    curLineLen += tok.length();
+                    if (curLineLen > 40) {
+                        curLineLen = 0;
+                    }
                 }
             }
-        }
-
 
-        String pString =
-            "\n-------------------------------------------------------\n" + // 50 chars
-            charName + "\n" +
-            "-------------------------------------------------------\n" + // 50 chars
-            "User:        " + username + "\n" +
-            "Story:       " + story + "\n" +
-            "Race:        " + race + "\n" +
-            "Archetype:   " + arch + "\n" +
-            "Party:       " + party + "\n" +
-            "Power:       " + power + "\n" +
-            "Proficiency: " + proficiency + "\n" +
-            "Personality: " + personality + "\n" +
-            "Percecption: " + percecption + "\n" +
-            "Experience:  " + experience + "\n" +
-            "-------------------------------------------------------\n";
-
-        System.out.println(pString);
+            String pString =
+                "\n-------------------------------------------------------\n" + // 50 chars
+                charName + "\n" +
+                "-------------------------------------------------------\n" + // 50 chars
+                "User:        " + username + "\n" +
+                "Story:       " + story + "\n" +
+                "Race:        " + race + "\n" +
+                "Archetype:   " + arch + "\n" +
+                (party != null? "Party:       " + party + "\n" : "") +
+                "Power:       " + power + "\n" +
+                "Proficiency: " + proficiency + "\n" +
+                "Personality: " + personality + "\n" +
+                "Perception:  " + perception + "\n" +
```

Comment:
I think we should leave it as is. It looks weird having them in all caps and experience by itself below.